src/components/Calendar.tsx

```diff
@@ -15,6 +15,7 @@ import { setupLocales } from '../helpers/calendarHelper';
 import { QUERY_TYPES, getQuery } from '../queries';
 import { ScreenName, Calendar as TCalendar } from '../types';
 
+import { normalize } from 'react-native-elements';
```

Comment:
please use our normalize and not the RNE one, right?!

Reply:
Yes, you're right. I missed it.
https://github.com/smart-village-solutions/smart-village-app-app/commit/f7d54af5ad1ed02cec8b2ffb76217bfd75f73029